src/components/TreeList/__stories__/TreeList.mdx

```diff
@@ -0,0 +1,315 @@
+import {Meta} from '@storybook/addon-docs';
+
+<Meta title="Unstable/TreeList" />
+
+# TreeList
+
+The basic component for working with lists, including tree-like ones. Under the hood, it uses the [useList](/docs/unstable-uselist--docs). To manage the state, it is recommended to use the [useListState](/docs/unstable-uselist--docs#useliststate) hook.
+
+`Storybook` provides complex examples how to use this components from this documentation.
+
+## Props:
+
+-   [items](#items);
+-   [mapItemDataToProps](#mapitemdatatoprops);
+-   [qa](#qa);
+-   [id](#id);
+-   [containerRef](#containerref);
+-   [className](#classname);
+-   [multiple](#multiple);
+-   [size](#size-available-options);
+-   [defaultGroupsExpanded](#defaultgroupsexpanded);
+-   [getId](#getid);
+-   [renderItem](#renderitem);
+-   [renderContainer](#rendercontainer);
+-   [onItemClick](#onitemclick);
+-   [...useListState](/docs/unstable-uselist--docs#useliststate)
+
+## Quick start:
+
+### Basic example:
+
+```tsx
+import {type ListItemType, getItemRenderState, TreeList} from '@gravity-ui/uikit';
+
+const items: ListItemType<string>[] = ['one', 'two', 'free', 'four', 'five'];
+
+<TreeList items={items} mapItemDataToProps={(title) => ({title})} />;
```

Comment:
I would name the parameter as `item` to emphasize that it is the element of items array